src/app/routes/app-routes.tsx

```diff
@@ -80,8 +83,9 @@ export const homePageModalRoutes = (
 function useAppRoutes() {
   return sentryCreateBrowserRouter(
     createRoutesFromElements(
-      <Route element={<Container />}>
-        <Route key="error" errorElement={<RouterErrorBoundary />}>
+      <Route>
```

Comment:
I think we still want a container/root component to attach global behaviours to, e.g. `useOnSignOut` etc? Maybe worth renaming to not confuse the different styles of "container"

Reply:
Thanks for the feedback, I will update the names when finalising. (I have a bit of refactoring + organising to do)

We do still have that root `<Container` for global behaviours which I pass a layout prop to to indicate which layout to use. You can see it [here](https://github.com/leather-io/extension/pull/5619/files#diff-550189a0c8f9f6f24b0af91a7e6c714c978d2935b94239cedb39bf11a23532e1) 

To compose routes, I need to wrap them  in a `parent `<Route`. 

```
<Route>
     <Route element={<Container layout={X}> 
      <!-- child routes --> 
     <Route/>
     <Route element={<Container layout={Y} > 
      <!-- child routes --> 
     <Route/>
 
 ```

Reply:
Got it, but why not have a single component that handles non-style related global behaviours (as before), rather than repeating it every time?

Some behaviours are genuinely global so feels weird to have to repeat every time, whereas _style_ changes we do want to be specific. Can't help but feel this design still kinda couples together global behaviours with component specific styles.

Why not

```
<Route element={<Container />}>
     <Route element={<LayoutX> 
      <!-- child routes --> 
     <Route/>
     <Route element={<LayoutY> 
      <!-- child routes --> 
     <Route/>
```

Reply:
OK, thanks, I'll de-couple as suggested 👍

Reply:
I've updated this now